packages/angular/tsup.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from "tsup";
+
+const supportedPackages = ['data-connect'];
+export default defineConfig({
+	entry: [`src/data-connect/index.ts`, 'src/index.ts'],
+	format: ["esm"],
+	dts: true, // generates .d.ts files
+	outDir: "./",
+	esbuildOptions(options, context) {
+		options.outbase = './src';
+	},
```

Comment:
To clean output directory before each build
```suggestion
	clean: true,
	},
```

Reply:
Keeping as-is until we resolve the module resolution discussion above. If I keep `clean: true`, it will delete the whole package.

Reply:
Updated.